libraries/libfc/test/crypto/test_webauthn.cpp

```diff
@@ -188,13 +188,12 @@ BOOST_AUTO_TEST_CASE(challenge_wrong_base64_chars) try {
    });
 } FC_LOG_AND_RETHROW();
 
-//valid signature but replace the padding '=' with '.'
+//valid signature but replace the last character '=' with '.'
 BOOST_AUTO_TEST_CASE(challenge_base64_dot_padding) try {
    webauthn::public_key wa_pub(pub.serialize(), webauthn::public_key::user_presence_t::USER_PRESENCE_NONE, "fctesting.invalid");
    std::string b64 = fc::base64url_encode(d.data(), d.data_size());
    char& end = b64.back();
 
-   BOOST_REQUIRE(end == '=');
    end = '.';
```

Comment:
This test was originally changing the padding from a valid character to an invalid character. Now it's changing a non-padding character from a valid character to an invalid character. I think maybe this should just append a `.` (or really any other invalid character) to keep the spirit of the original test; we already have `challenge_non_base64` to cover validity of non-padded characters.